deploy/helm/values.yaml

```diff
@@ -0,0 +1,39 @@
+enable_metrics:
+  - perf_event
```

Comment:
should we enable these by default? maybe they should be opt-in similar as normal cAdvisor?

Reply:
Yes, I agree, they should be opt-in similar to normal cAdvisor.